python-ovirt-engine-sdk4.spec.in

```diff
@@ -27,16 +27,47 @@ Requires: python3-six
 This package contains the Python 3 SDK for version 4 of the oVirt Engine
 API.
 
+%if %{rhel} < 9
+%package -n python38-ovirt-engine-sdk4
+Summary: oVirt Engine Software Development Kit (Python)
+BuildRequires: python38-devel
+Requires: libxml2
+Requires: python38
+Requires: python38-pycurl >= 7.43.0-6
+Requires: python38-six
+
+%description -n python38-ovirt-engine-sdk4
+This package contains the Python 3 SDK for version 4 of the oVirt Engine
+API.
+
+%files -n python38-ovirt-engine-sdk4
```

Comment:
To make the spec more readable wouldn't it be better to move 38 files section after normal files section?